docs/cmd/kn_service_create.md

```diff
@@ -58,6 +58,7 @@ kn service create NAME --image IMAGE [flags]
       --requests-cpu string      The requested CPU (e.g., 250m).
       --requests-memory string   The requested memory (e.g., 64Mi).
       --revision-name string     The revision name to set. Must start with the service name and a dash as a prefix. Empty revision name will result in the server generating a name for the revision. Accepts golang templates, allowing {{.Service}} for the service name, {{.Generation}} for the generation, and {{.Random [n]}} for n random consonants. (default "{{.Service}}-{{.Random 5}}-{{.Generation}}")
+      --service-account string   Service account name to set. Empty service account name will result to clear the service account.
```

Comment:
I know, it's a bit nit-picky, but using a `--service-account` without arg during a `kn service create` is a no-op and it won't clear the service account (as there is one). I know that this option is shared, but maybe we can use still different help messages for create & update ?

happy to get this PR merged now, but maybe we can adjust this in a later PR.